cli/templateinit.go

```diff
@@ -14,7 +17,8 @@ import (
 )
 
 func (*RootCmd) templateInit() *clibase.Cmd {
-	return &clibase.Cmd{
+	var templateIDArg string
```

Comment:
Nit (minor): Calling this `arg` doesn't seem necessary and I don't think we do it in other places.